### PR TITLE
Insurrection gamemode

### PR DESCRIPTION
**Pull request by ninjacha**
essentially reskinned nukeops for now.

### CODE REVIEW

File: code/__defines/gamemode.dm [hunk 1]
```diff
@@ -24,6 +24,8 @@
 #define BE_PLANT      0x1000
 #define BE_MUTINEER   0x2000
 #define BE_PAI        0x4000
+#define BE_INNIE      0x5000
```
Comment (bloxgate) [inline code/__defines/gamemode.dm]: This should be 0x8000

**Comment by bloxgate** [comment]
Forcing travis to start building again

**Comment by caelaislinn** [comment]
https://github.com/ninjacha/HaloSpaceStation13/pull/3

**Comment by ninjacha** [comment]
git was being weird and wanted another update

cue errors.

**Pull request merged**